### ISSUE TITLE
Discussion: ny terminology i stedet for "Handel"

### ISSUE THREAD

**Issue by dybber** [DatalogiForAlle/MarketSim#119]
Jeg tror måske vi skal finde på nogle andre termer for "Din næste handel", "Lav din handel!" på spillersiden. Men kun måske.

Det handler jo om hvor meget man vil producere og hvilken pris man vil sælge, selve handlerne kan man jo ikke styre? Men det er dejlig simpelt sådan som det er. Det bliver mere abstrakt hvis der står "Næste runde", "Foretag valg" eller lignende intetsigende tekster.

**Comment by KristianMSchmidt** [comment]
Ja, jeg har tænkt over præcis det samme, men jeg har haft svært ved at finde på gode alternativer.

**Comment by dybber** [comment]
Vi tager den med til Niels

**Comment by dybber** [comment]
Jeg brainstormer lige:

"Lav din handel!" --> "Ready/Klar"?
"Lav din handel!" --> "Meld klar"
"Lav din handel!" --> "Sæt dine varer til salg"?
"Lav din handel!" --> "Næste runde"
"Lav din handel!" --> "Udbyd dine varer til salg"
"Lav din handel!" --> "Udbyd til valgt pris"
"Lav din handel!" --> "Udbyd til salg"
"Lav din handel!" --> "Sæt til salg"
"Lav din handel!" --> "Afhænd varer"

"Din næste handel" --> "Næste runde"
"Din næste handel" --> "Hvad er dit udbud?"
"Din næste handel" --> "Gør klar til næste runde"
"Din næste handel" --> "Gør klar til næste runde"
"Din næste handel" --> "Vælg pris og produktiontal til næste runde"
